iXBRLViewerPlugin/viewer/src/js/fact.js

```diff
@@ -262,31 +266,49 @@ export class Fact {
         return this.f.err == 'INVALID_IX_VALUE';
     }
 
+    getAccuracyLabel(value, isScale=false) {
+        return this._report.getAccuracyLabel(
+                isScale ? -value : value, // Invert value if this is a scale value
+                this.isMonetaryValue(),
+                this.report().qname(this.measure()).localname
+        );
+    }
+
     readableAccuracy() {
         if (!this.isNumeric() || this.isNil()) {
             return i18next.t("common.notApplicable");
         }
-        let d = this.decimals();
+        const d = this.decimals();
         if (d === undefined) {
             return i18next.t("common.accuracyInfinite")
         }
         else if (d === null) {
             return i18next.t("common.unspecified");
         }
-        var name = i18next.t(`currencies:accuracy${d}`, {defaultValue:"noName"});
-        if (this.isMonetaryValue()) {
-            var currency = this.report().qname(this.measure()).localname;
-            if (d == 2) {
-                var name = i18next.t(`currencies:cents${currency}`, {defaultValue: name});
-            }
+        const label = this.getAccuracyLabel(d);
+        if (label != null) {
+            return `${d} (${label})`;
```

Comment:
I'd prefer not to include the actual value of the scale attribute, or at least make it less prominent, as I think it's a technical detail, and it's meaning isn't obvious if you're not familiar with iXBRL.

I'd prefer to just say:

> Scale: millions

If it's important to include the value, perhaps do:

> Scale: millions (scale=6)

or just:

> Scale: millions (6)

Reply:
Agreed. Do you feel the same about the accuracy label?

Reply:
Yes, I think so.  I think the number is slightly more meaningful for accuracy, as I think "accurate to N decimal places" is more understandable than "scaled by 10 to the N", but on balance I think we should swap it.

Maybe we could do it like this:

> Scale: millions (scale=6)
> Accuracy: millions (decimals=-6)

Which makes the "business" meaning primary, but keeps the XBRL technical detail for those who understand it.  Or possibly we could put the parenthetical bit in a tooltip.

Reply:
I like the tooltip to keep clutter low. Perhaps that's something we can incorporate if we implement some kind of abstraction for rending fact details to easily configure tooltips for each detail.